tasks/buildah.yaml

```diff
@@ -17,7 +17,7 @@ spec:
     description: The location of the buildah builder image.
     name: BUILDER_IMAGE
     type: string
-  - default: vfs
+  - default: overlay
```

Comment:
Reason of changing this? Overlay does not work in openshift (as you cans see in CI).

Reply:
Because when I was using vfs, the  image was not being found. Probably because my environment is a Kind cluster :grimacing: 
I was about to test it in a real cluster berfore marking the PR as ready to review.

Reply:
yeah only privileged or run as root users can use overlay

Reply:
The error was that I was not setting the flag in the push command. It is working now.